styles.css

```diff
@@ -55,12 +53,17 @@ img {
     * {
         font-size: 1rem;
     }
+
+    .contents {
+        width: 400px;
+    }
+
     main > section {
         width: 400px;
```

Comment:
similarly - perhaps a percentage would be better

Reply:
done